lib/services/settings_service.dart

```diff
@@ -8,7 +10,11 @@ class SettingsService {
   static Box<SidekickSettings> box;
 
   static Future<void> init() async {
-    box = await Hive.openBox<SidekickSettings>(key);
+    try {
+      box = await Hive.openBox<SidekickSettings>(key);
+    } on FileSystemException {
+      exit(0);
```

Comment:
@aguilaair This issue for me happens when I rerun sidekick and 2 instances are running. Should this only happen in debugging? If so should we throw an exception that tells the dev/user to close another instance of the sidekick. The exit code 0 is success, should this be the case?

Reply:
Not exactly, In windows you can spawn multiple entities:
![image](https://user-images.githubusercontent.com/29983481/116829406-32312580-aba4-11eb-9ebf-ba9068b89817.png)

Reply:
Regarding the exit code, I was also thinking about it. In any case, we aren't able to open the db file multiple times and as long as we have that in mind I would consider the exit as graceful because we are having this in mind. Alternatively, we could show some kind of warning, but this seemed like the easiest.

Reply:
I think its important to throw an error, for example when this happens for me I notice because of the error thrown. Either we should show a notice, throw an error

Reply:
sounds good, we can do that then

Reply:
Added
![image](https://user-images.githubusercontent.com/29983481/116876568-fc7c5300-ac1c-11eb-86c2-d97fe9362819.png)